### ISSUE TITLE
Improve Documentation: Add MutliOptimizer documentation to RTD

### ISSUE THREAD

**Issue by avital** [google/flax#788]

**Comment by marcvanzee** [comment]
The canonical example is as follows:

```
kernels = optim.ModelParamTraversal(lambda path, _: 'kernel' in path)
biases = optim.ModelParamTraversal(lambda path, _: 'bias' in path)
kernel_opt = optim.Momentum(learning_rate=0.01)
bias_opt = optim.Momentum(learning_rate=0.1)
opt_def = MultiOptimizer((kernels, kernel_opt), (biases, bias_opt))
optimizer = opt_def.create(model)
```

If you want to update the learning rates of both optimizers online with different learning rate schedules, you should use the following code:

```
new_opt=optimizer.apply_gradient(
    grads, 
    hyper_params=[
      optimizer.optimizer_def.hyper_params[0].replace(learning_rate=0.2), 
      optimizer.optimizer_def.hyper_params[1].replace(learning_rate=0.4),
    ])
```

**Comment by marcvanzee** [comment]
@jheek mentioned that we actually have a good explanation in the docstring of MultiOptimizer: https://github.com/google/flax/blob/master/flax/optim/base.py#L397

However, it does not appear on readthedocs, so I am renaming this issue accordingly.
